odoo_tools/cli/project.py

```diff
@@ -302,6 +303,19 @@ def checkout_local_odoo(
     else:
         ui.exit_msg("Unable to find the commit hash of odoo enterprise")
 
+    # Apply odoo/enterprise patches
+    # This matches the behavior of the `001_apply_patches` script in the odoo-template
+    # https://github.com/camptocamp/odoo-template/blob/99931bbc/%7B%7Bcookiecutter.repo_name%7D%7D/build.d/001_apply_patches
+    patches_dir = build_path("./patches")
+    if patches_dir.exists() and patches_dir.is_dir():
```

Comment:
Second test is enough:

```suggestion
    if patches_dir.is_dir():
```